ktor-shared/ktor-serialization/ktor-serialization-jackson/jvm/src/JacksonConverter.kt

```diff
@@ -95,6 +121,59 @@ public class JacksonConverter(
             }
         }
     }
+
+    private companion object {
+        private const val beginArrayCharCode = '['.code
+        private const val endArrayCharCode = ']'.code
+        private const val objectSeparator = ','.code
+    }
+
+    /**
+     * Guaranteed to be called inside a [Dispatchers.IO] context, see [OutputStreamContent]
+     */
+    @Suppress("BlockingMethodInNonBlockingContext")
+    private suspend fun <T> Flow<T>.serializeJson(outputStream: OutputStream) {
+        val jfactory = JsonFactory()
+        // cannot use ObjectMapper write to Stream because it flushes the OutputStream on each write
```

Comment:
Because the normal behavior of `ObjectMapper` is to flush every time a Jackson write* that targets the IO Writer or OutputStream is called (= the value in the Flow), that's not what I want, I want to flush only after the separator is written.